src/private/utils.ts

```diff
@@ -30,19 +30,7 @@ export function decorate(handleDescriptor, entryArgs) {
 }
 
 class Meta {
-  @lazyInitialize
-  debounceTimeoutIds = {};
-
-  @lazyInitialize
-  throttleTimeoutIds = {};
-
-  @lazyInitialize
-  throttlePreviousTimestamps = {};
-
-  @lazyInitialize
-  throttleTrailingArgs = null;
-
-  @lazyInitialize
+  // @lazyInitialize
```

Comment:
sorry if I'm misunderstanding but it's still not clear if this comment extraneous? It seems like it is